OpenRA.Game/Network/SyncReport.cs

```diff
@@ -141,12 +147,15 @@ internal void DumpSyncReport(int frame)
 					Log.Write("sync", "Orders Issued:");
 					foreach (var o in r.Orders)
 						Log.Write("sync", "\t {0}", o.ToString());
-
-					return;
 				}
 			}
 
-			Log.Write("sync", "No sync report available!");
+			Log.Write("sync", "Sync Report System Info:");
+			Log.Write("sync", "Out of sync frame: {0}", frame);
+			Log.Write("sync", "Recorded frames: " + string.Join(",", recordedFrames));
+
+			if (!desyncFrameFound)
+				Log.Write("sync", "Recorded frames do not contain the frame {0}. No sync report available!", frame);
```

Comment:
```suggestion
				Log.Write("sync", "Recorded frames do not contain the frame {frame}. No sync report available!");
```

We are switching to inline variables in the code base. It would be great to do it here as well. Preferably as a separate commit

Reply:
You need a $ before the string. Changed anyway.